app/javascript/articles/components/CommentsCount.jsx

```diff
@@ -26,6 +26,7 @@ export const CommentsCount = ({ count, articlePath, articleTitle }) => {
         icon={commentsSVG}
         tagName="a"
         aria-label={commentsAriaLabelText}
+        tooltip={`Add a comment to post - ${articleTitle}`}
```

Comment:
```suggestion
        aria-label={`Add a comment to post - ${articleTitle}`}
```

I don't think it's part of the requirements to introduce a tooltip, but we do want the accessible name to reflect the action that's going to happen on click, so let's just switch these out 🙂